core/src/test/java/org/infinispan/configuration/parsing/UnifiedXmlFileParsingTest.java

```diff
@@ -523,11 +523,6 @@ private static void configurationCheck70(EmbeddedCacheManager cm) {
       assertTrue(c.dataContainer().dataContainer() instanceof QueryableDataContainer);
       assertTrue(c.dataContainer().<byte[]>keyEquivalence() instanceof AnyEquivalence);
       assertTrue(c.dataContainer().<byte[]>valueEquivalence() instanceof AnyEquivalence);
-
-      c = cm.getCacheConfiguration("store-as-binary");
```

Comment:
We should verify that BINARY storage type has been enabled instead of just removing.